CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to JokesOnYou
+A big welcome and thank you for considering contributing to **JokesOnYou**, Here we create website where people can view and rate jokes.
+We formed this project to learn about web development using C# as main Language.
+This should be a learning project for all aspects. The Building, Coding, Documenting and Teaching.
+It’s people like you that make it a reality for users in our community.
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## Pull Requests
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `main`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## We follow the "fork-and-pull" Git workflow
+
```

Comment:
This looks awesome man! Could we also bring up that we prefer if they become a part of our discord so we can easily communicate and help each other out?

Reply:
well it looks fine but I don't track discord often tbh 😂.
I'll try to be active.

Thanks for asking